clone-master-clean-up.sh

```diff
@@ -230,28 +255,30 @@ if [ "$CMCU_USERIDS" = "yes" ]; then
     done
 fi
 
-echo "swap the uuid strings with dev strings in /etc/fstab"
-> /tmp/fstab.tmp
-while read -r disk remain; do
-    case "$disk" in
-    UUID=*)
+if [ "${DONT_CHANGE_FSTAB}" != "YES" ]; then
+    echo "swap the uuid strings with dev strings in /etc/fstab"
+    > /tmp/fstab.tmp
+    while read -r disk remain; do
+        case "$disk" in
+        UUID=*)
         uuid=${disk#UUID=}
         new_disk=$(/usr/sbin/blkid -U "$uuid")
-        ;;
-    LABEL=*)
-        label=${disk#LABEL=}
-        new_disk=$(/usr/sbin/blkid -L "$label")
-        ;;
-    *)
+    ;;
+       LABEL=*)
+       label=${disk#LABEL=}
+       new_disk=$(/usr/sbin/blkid -L "$label")
+    ;;
+        *)
         new_disk="$disk"
-        ;;
-    esac
-    echo "$new_disk $remain" >> /tmp/fstab.tmp
-done < /etc/fstab
-if [ -s /tmp/fstab.tmp ]; then
-    cp /tmp/fstab.tmp /etc/fstab
+    ;;
+        esac
```

Comment:
starting line 263 the indentation is a little bit confusing. Can we please get an easier to read case-statement

Reply:
Wo du Recht hast, da hast du Recht.